sdk/pchain/prepare_create_chain_tx.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package pchain
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/sdk/multisig"
+	"github.com/ava-labs/avalanche-cli/sdk/wallet"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+)
+
```

Comment:
thinking this should be on wallet

Reply:
https://github.com/ava-labs/avalanche-tooling-sdk-go/blob/fef92d1127dc37da50a9307a78663b81a04b971e/wallet/p-chain/txs/convertSubnetToL1Tx.go